SYCL/Regression/context_with_multiple_devices.cpp

```diff
@@ -0,0 +1,39 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// REQUIRES: level_zero && gpu
+
+// Test the Level Zero backend with context having multiple root devices
+
+#include <CL/sycl.hpp>
+
+using namespace cl::sycl;
+
+int main() {
+  auto Plt = platform(gpu_selector{});
+  auto Devs = Plt.get_devices();
+  auto Ctx = context(Devs);
+
+  // Test the patform with 2 or more devices
+  if (Devs.size() < 2) {
```

Comment:
In most configuration only one GPU device is available. Moreover when we have 2 GPU devices they can be in different platforms. So the test will never get expected configuration..
Will the test work if we create sub-devices to make sure that we have proper configuration?

Reply:
Could you try to use NEOReadDebugKeys=1 CreateMultipleRootDevices=2?
```
$ NEOReadDebugKeys=1 CreateMultipleRootDevices=2 sycl-ls
[opencl:0] GPU : Intel(R) OpenCL HD Graphics 3.0 [21.19.19792]
[opencl:1] GPU : Intel(R) OpenCL HD Graphics 3.0 [21.19.19792]
[opencl:0] CPU : Intel(R) OpenCL 2.1 [2021.11.3.0.09_160000]
[opencl:0] ACC : Intel(R) FPGA Emulation Platform for OpenCL(TM) 1.2 [2021.11.3.0.09_160000]
[level_zero:0] GPU : Intel(R) Level-Zero 1.1 [1.1.19792]
[level_zero:1] GPU : Intel(R) Level-Zero 1.1 [1.1.19792]
[host:0] HOST: SYCL host platform 1.2 [1.2]
```

Reply:
@againull could you follow up with the PR?

Reply:
Fixed.